index.html

```diff
@@ -11119,9 +11119,10 @@ <h2>Type Maps</h2>
       or an array containing both <code>@type</code> and <code>@set</code>.
       The keys of a <a>type map</a> MUST be <a>IRIs</a>
       (<a>relative IRI</a>, <a>compact IRI</a> (including <a>blank node identifiers</a>), or <a>absolute IRI</a>),
-      the <a>keyword</a> <code>@none</code>,
-      or a <a>term</a> which expands to <code>@none</code>,
-      and the values MUST be <a>node objects</a>.</p>
+      <a>terms</a>,
```

Comment:
the previous version limited this to "terms expaning to @none", but for type-map, it seems to me that any term can be used (at least, this works in the playground and the Ruby distiller)

Reply:
This line is in the Type Maps section, which has always allowed terms.